roles/etcd/tasks/upgrade_static.yml

```diff
@@ -0,0 +1,45 @@
+---
+# PREREQ Node service is ready to run static pods
+
+# INPUT r_etcd_upgrade_version
+- name: Verify cluster is healthy pre-upgrade
+  command: "{{ etcdctlv2 }} cluster-health"
+
+- name: Remove old etcd service files
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+  - "/etc/systemd/system/etcd.service"
+  - "/etc/systemd/system/etcd_container.service"
+
+# We removed the ability to detect what was previously 'containerized'
+# Need to stop and disable this service, but might not be present.
+- name: Stop, disable old etcd services
+  systemd:
+    name: "{{ item }}"
+    state: stopped
+    enabled: no
+  failed_when: False
+  with_items:
+  - etcd
+  - etcd_container
+
+- name: Mask old etcd services
+  command: "systemctl mask {{ etcd_service }}"
+  with_items:
+  - etcd
+  - etcd_container
+
+- name: Reload systemd daemon
+  command: "systemctl daemon-reload"
+
+- name: Configure static definition
+  import_tasks: static.yml
+
+- name: Verify cluster is healthy
```

Comment:
@sdodson I was under the impression that this task was in charge of waiting for etcd to be ready.  At least that's how it's done for the other upgrade paths?